homeassistant/components/asuswrt/bridge.py

```diff
@@ -29,16 +32,25 @@
     DEFAULT_INTERFACE,
     KEY_METHOD,
     KEY_SENSORS,
+    PROTOCOL_HTTP,
+    PROTOCOL_HTTPS,
     PROTOCOL_TELNET,
     SENSORS_BYTES,
+    SENSORS_CPU,
     SENSORS_LOAD_AVG,
+    SENSORS_MEMORY,
     SENSORS_RATES,
     SENSORS_TEMPERATURES,
+    SENSORS_TEMPERATURES_LEGACY,
 )
 
+HTTP_PROTOCOLS = (PROTOCOL_HTTPS, PROTOCOL_HTTP)
+
 SENSORS_TYPE_BYTES = "sensors_bytes"
 SENSORS_TYPE_COUNT = "sensors_count"
+SENSORS_TYPE_CPU = "sensors_cpu"
```

Comment:
These are new sensors introduced? That feels out of scope for this PR.

Reply:
Removed new sensors, I will add in a new PR after this will be completed.